spacy/tests/test_cli.py

```diff
@@ -855,3 +857,126 @@ def test_span_length_freq_dist_output_must_be_correct():
     span_freqs = _get_spans_length_freq_dist(sample_span_lengths, threshold)
     assert sum(span_freqs.values()) >= threshold
     assert list(span_freqs.keys()) == [3, 1, 4, 5, 2]
+
+
+def test_cli_find_threshold(capsys):
+    thresholds = numpy.linspace(0, 1, 10)
+
+    def make_examples(_nlp: Language) -> List[Example]:
+        docs: List[Example] = []
+
+        for t in [
+            (
+                "I am angry and confused in the Bank of America.",
+                {
+                    "cats": {"ANGRY": 1.0, "CONFUSED": 1.0, "HAPPY": 0.0},
+                    "spans": {"sc": [(31, 46, "ORG")]},
+                },
+            ),
+            (
+                "I am confused but happy in New York.",
+                {
+                    "cats": {"ANGRY": 0.0, "CONFUSED": 1.0, "HAPPY": 1.0},
+                    "spans": {"sc": [(27, 35, "GPE")]},
+                },
+            ),
+        ]:
+            doc = _nlp.make_doc(t[0])
+            docs.append(Example.from_dict(doc, t[1]))
+
+        return docs
+
+    def init_nlp(
+        components: Tuple[Tuple[str, Dict[str, Any]], ...] = ()
+    ) -> Tuple[Language, List[Example]]:
+        _nlp = English()
```

Comment:
I don't think there's any reason for private-looking names in this method with both `_nlp` and `_examples`:

```suggestion
        nlp = English()
```

Reply:
Not strictly, but it would shadow the variable names `nlp` and `examples` in the outer scope. Renamed it to `new_nlp` and `new_examples` - does that make more sense?